mission-board/src/main/java/com/missionBoard/board/controller/BoardController.java

```diff
@@ -0,0 +1,57 @@
+package com.missionBoard.board.controller;
+
+import com.missionBoard.board.domain.Board;
+import com.missionBoard.board.dto.BoardRequest;
+import com.missionBoard.board.dto.BoardResponse;
+import com.missionBoard.board.service.BoardService;
+import com.missionBoard.config.LoginUser;
+import com.missionBoard.login.domain.Member;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/boards")
+public class BoardController {
+
+    private final BoardService boardService;
+
+    public BoardController(BoardService boardService) {
+        this.boardService = boardService;
+    }
+
+    @PostMapping("/create")
```

Comment:
```suggestion
    @PostMapping
```

RESTful에 의해 POST /api/boards 라고 해도 충분할 것 같습니다!